### PR TITLE
LPS-193747 Replace AUI with js/react in server-admin-web module

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
The motivation behind this ticket is to migrate from AUI old javascript files to functional/react js components.
https://liferay.atlassian.net/browse/LPS-193747

Proposed Solution
========
What I did to solve this problem is to change the file structure and to use equivalents methods in new js structure

Steps to Verify
========
1. Go to Server Administration and check that all the action buttons work correctly

### CODE REVIEW

File: modules/apps/server/server-admin-web/package.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+	"dependencies": {
+		"frontend-js-web": "*"
+	},
```
Comment (victorg1991) [inline modules/apps/server/server-admin-web/package.json:4]: I believe this is not needed

File: modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/js/main.js [hunk 2]
```diff
@@ -3,135 +3,44 @@
...
+export default function ({namespace: portletNamespace, redirectURL, url}) {
+	const form = document.getElementById(`${portletNamespace}fm`);
+
+	const addInputsFromData = (data) => {
+		const nsKey = Liferay.Util.ns(portletNamespace, 'cmd');
+		const input = (
+			<input id={nsKey} name={nsKey} type="hidden" value={data.cmd} />
+		);
+		form.append(input);
+	};
```
Comment (victorg1991) [inline modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/js/main.js:18]: Mmm This is a bit weird, we are generating JSX and pass it to the html form component. I think this is not working as it should. Also in the AUI component we were iterating over all the data attributes but here we are only taking the cmd value

Can you tweak this a bit? For adding an input to a form we should be creating the input directly without needing react. Something like:

```javascript
const input = document.createElement('input')

input.name = 'watevername'
input.id = 'wateverId'
input.value = 
input.type = 'hidden'
```

Also I believe we don't need to use the `ns` utility in this case, we can just do a string interpolation like ` ${portletNamespace}name`

**Comment by victorg1991** [comment]
Hey @Stephy6 can you take a look at the comments :innocent: Let me know if you need any help :)

**Comment by victorg1991** [comment]
ci:forward

**Comment by victorg1991** [comment]
Nice @Stephy6 👏

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**